docs/how-tos/validate-nexus-file.md

```diff
@@ -0,0 +1,802 @@
+
+The NeXus file is valid if it complies with the respective NeXus application definition.
+
+This validation is done by software.
+
+
+# Validation software
+
+There are right now three tools, which can be used for validation of NeXus files. All are different and have individual advantages or disadvantages:
+
+1. pynxtools 
+
+2. cnxvalidate
+
+3. punx
+
+Open software is usually shared on Github - There you find usually the most accurate information, as documentation sometimes lags behind. There you see a box with folders and files. Below is the content of the README.md file displayed. This usually shows instructions for installation and handling of the software.
+
+Here are the GitHub links for the thee software packages:
+
+[pynxtools](<https://github.com/FAIRmat-NFDI/pynxtools>)
+
+[cnxvalidate](<https://github.com/nexusformat/cnxvalidate>)
+
+[punx](<https://github.com/prjemian/punx>)
+
+In the following, each package and its capabilities is presented.
+
+### Operating systems
+
+Almost all PC users are used to Windows as operating system.
+
+A lot of software development is done on Linux as operating system.
+
+This is not a problem for big companies, but for smaller open software projects, which are often developed without funding, this is a problem.
+
+If you are used to Windows, consider setting up a Linux operating system to eliminate problems in the installation process and ensure compatibility.
+
+
+# 1. pynxtools - Python Nexus Tools
+
+This is a python package which is developed by the FAIRmat consortium.
+
+As a python package, this can be used on Linux and Windows systems.
+
+The package can be installed via pip. Therefore you need to have installed:
+
+1. python
+
+2. pip
+
+This tool has 3 command line functions:
+
+1. [dataconverter](<https://github.com/FAIRmat-NFDI/pynxtools/blob/master/src/pynxtools/dataconverter/README.md>)
+
+2. [read\_nexus](<https://github.com/FAIRmat-NFDI/pynxtools/blob/master/src/pynxtools/nexus/README.md>)
+
+3. [generate\_eln](<https://github.com/FAIRmat-NFDI/pynxtools/blob/master/src/pynxtools/eln_mapper/README.md>)
+
+4. [verify\_nexus](https://github.com/FAIRmat-NFDI/pynxtools/pull/333)
+
+
+For validation purposes, we will use the "read\_nexus" and "verify\_nexus" function.
+
+# 1.1 verify_nexus
+
+This tool is currently in development. It enables a command like:
+
+```
+verify_nexus C:\nexusvalidation\Raman.nxs
+```
+
+
+The output warning looks like this:
+```
+...
+WARNING: Field /entry/instrument/beam_incident/wavelength/@units written without documentation.
+...
+```
+
+# 1.1.1 Installation of verify_nexus
+
+Aside from read_nexus, there is a second functionality from pynxtools. This method is currently in development (Aug 2024). This function is called: verify\_nexus. Therefore, you have to use the development install, until this function is published.
+
```

Comment:
You have not mentioned read_nexus so far, so doesn't make sense to talk about a second functionalit
```suggestion
The `verify_nexus` function is currently under development (Aug 2024). Therefore, you have to install pynxtools from its feature branch until this function is published.
```